pkg/builders/catalogsourcebuilder.go

```diff
@@ -62,10 +62,14 @@ func (b *CatalogSourceBuilder) WithOLMLabels(cscLabels map[string]string) *Catal
 }
 
 // WithOwnerLabel sets the owner label of the CatalogSource object to the given owner.
-func (b *CatalogSourceBuilder) WithOwnerLabel(name, namespace string) *CatalogSourceBuilder {
-	labels := map[string]string{
-		OwnerNameLabel:      name,
-		OwnerNamespaceLabel: namespace,
+func (b *CatalogSourceBuilder) WithOwnerLabel(name, namespace string, isOpsrcOwner bool) *CatalogSourceBuilder {
+	labels := map[string]string{}
+	if isOpsrcOwner {
+		labels[OpsrcOwnerNameLabel] = name
+		labels[OpsrcOwnerNamespaceLabel] = namespace
+	} else {
+		labels[OwnerNameLabel] = name
+		labels[OwnerNamespaceLabel] = namespace
```

Comment:
This code is reused in multiple places. Could we create `getOpsrcOwnerLables` and `getCscOwnerLabels` functions that return the labels map? That way, if the labels change in the future we can address this all in one spot.

Reply:
Good suggestion! What do you think @kevinrizza @aravindhp @anik120 ?

Reply:
Alternatively you could create a `getLabels` method that takes the `isOpsrcOwner` bool and returns the correct labels - that way you don't add  `if/else` statements everywhere.